base/array.jl

```diff
@@ -1835,22 +1835,60 @@ function empty!(a::Vector)
 end
 
 _memcmp(a, b, len) = ccall(:memcmp, Cint, (Ptr{Cvoid}, Ptr{Cvoid}, Csize_t), a, b, len % Csize_t) % Int
```

Comment:
`bitset.jl` and `parse.jl` seem to be the only remaining users of this, and look trivial to change to `memset`?